libs/perun/components/src/lib/selection-item-search-select/selection-item-search-select.component.ts

```diff
@@ -153,7 +144,7 @@ export class SelectionItemSearchSelectComponent implements OnInit {
     this.item.value = this.selectedAttribute;
   }
 
-  sortAttributes(attributes: SelectionItem[]) {
+  sortAttributes(attributes: SelectionItem[]): void {
     attributes.sort((a: SelectionItem, b: SelectionItem) => {
```

Comment:
I think there should be also return type.
```suggestion
  sortAttributes(attributes: SelectionItem[]): void {
    attributes.sort((a: SelectionItem, b: SelectionItem): number => {
```